config/default.js

```diff
@@ -356,6 +356,9 @@ module.exports = {
   // We don't have to keep it secret, we use this JWT token just to compress data, not to secure it
   NYLAS_CONNECT_CALENDAR_JWT_SECRET: process.env.NYLAS_CONNECT_CALENDAR_JWT_SECRET || 'secret',
 
+  // Work type ID used with standardised skills.  This ID corresponds to a gig / job:
+  STANDARDIZED_SKILL_WORK_TYPE_ID: '413340e8-f56f-4061-8796-c916dfa82770',
```

Comment:
Will this be same id in prod as it is hard coded here?

Reply:
@kkartunov , @jmgasper ,
This is Chris call, he is setting up the Production database with the data.
Once we get access to Prod db we can check its value

Reply:
@kkartunov , @jmgasper ,
I accessed the Prod database and can confirm that this is the same id used for gig source type:
![prod-gig-workType](https://github.com/topcoder-platform/taas-apis/assets/5437419/26a4f209-aa3b-4136-ad9c-4c23b408c231)